INN/umbrella-sfpublicpress
# Styling for related posts widget

## Changes

This pull request makes the following changes:

- Adds styling for the Largo Related Posts widget

Desktop:
![Screen Shot 2020-02-11 at 4 35 46 PM](https://user-images.githubusercontent.com/18353636/74281785-069af480-4ced-11ea-9c94-e8eb66f38584.png)

Mobile:
![screencapture-sfpp-test-2020-02-11-16_37_35](https://user-images.githubusercontent.com/18353636/74281777-03a00400-4ced-11ea-8299-4dd3d2d3e521.png)

## Why



For #31

## Testing/Questions

Features that this PR affects:

- Largo Related Posts widget


Questions that need to be answered before merging:

- [ ] The images look way blurry in this with `width: 100%;` since the actual 120x120 thumbnail is being pulled in. @benlk do you have a good idea for how we can grab a larger image size here since there's no widget option for it?

Steps to test this PR:

1. Add the Largo Related Posts widget to the article bottom widget area with these settings:
   - Title: Related Stories
   - Posts: 3
   - Show byline: yes
   - Widget title link: `#`
2. View an article on desktop and mobile and make sure the styling matches the designs
